miso-web/src/main/webapp/scripts/hot_boxable.js

```diff
@@ -118,58 +127,69 @@ HotTarget.boxable = (function() {
                 obj.box.id = null;
               }
             },
-            depends: ['boxSearch', 'distributed'],
+            depends: ['boxSearch', 'distributed', 'discarded'],
             update: function(obj, flat, flatProperty, value, setReadOnly, setOptions, setData) {
               var applyChanges = function(source, autoSelect) {
                 setOptions({
                   source: source
                 });
+
                 setData(source.length > 1 && !autoSelect ? 'SELECT' : source[0]);
               };
-              if (!value || (flatProperty == 'distributed' && value == 'Sent Out')) {
+              if (!value) {
                 applyChanges([''], false);
                 return;
               }
+              if ((flatProperty === 'discarded') || (flatProperty === 'distributed')) {
+                applyChanges([''], true);
+                return;
+              }
               if (boxSearchCache[value.toLowerCase()]) {
                 applyChanges(boxSearchCache[value.toLowerCase()].map(function(item) {
                   return item.alias;
-                }), (boxSearchCache[value.toLowerCase()][0].name.toLowerCase() == value.toLowerCase()
-                		|| boxSearchCache[value.toLowerCase()][0].alias.toLowerCase() == value.toLowerCase() 
-                		|| (boxSearchCache[value.toLowerCase()][0].identificationBarcode && boxSearchCache[value.toLowerCase()][0].identificationBarcode.toLowerCase() == value.toLowerCase())));
+                }),
+                    (boxSearchCache[value.toLowerCase()][0].name.toLowerCase() == value.toLowerCase()
+                        || boxSearchCache[value.toLowerCase()][0].alias.toLowerCase() == value.toLowerCase() || (boxSearchCache[value
+                        .toLowerCase()][0].identificationBarcode && boxSearchCache[value.toLowerCase()][0].identificationBarcode
+                        .toLowerCase() == value.toLowerCase())));
                 return;
               }
 
               setData('(...searching...)');
               var deferred = jQuery.Deferred();
-              jQuery.ajax({
```

Comment:
No actual code changes from here on down, Eclipse just decided to reformat things.